cabal.project

```diff
@@ -1,6 +1,10 @@
 index-state: 2023-10-29T16:11:26Z
 
-packages: .
+packages:
+    symbolic-base
+    symbolic-cardano
+    symbolic-examples
```

Comment:
Do `symbolic-examples` have to be a separate package from `symbolic-base`? I'd prefer to keep them as part of `base` to have the compiler benchmark closer to the `ArithmeticCircuit` definition

Reply:
We currently have one example from `symbolic-cardano`.

Reply:
The `symbolic-examples` package and `symbolic-base` package appear to have duplicates between them in `symbolic-base/examples/` and `symbolic-examples/src`.

Reply:
Thanks, fixed.